termine/announce.go

```diff
@@ -31,18 +33,7 @@ func init() {
 	cmdAnnounce.Run = RunAnnounce
 }
 
-func isStammtisch(date time.Time) (stammt bool, err error) {
-	err = db.QueryRow("SELECT stammtisch FROM termine WHERE date = $1", date).Scan(&stammt)
-	return
-}
-
-func announceStammtisch(date time.Time) {
-	loc, err := getLocation(date)
-	if err != nil {
-		log.Println("Kann Location nicht auslesen:", err)
-		return
-	}
-
+func announceStammtisch(t *data.Termin) error {
```

Comment:
`t *date.Termin`: one character-variables are easy to write, but having an actual word is better for later when someone works into the code.

Therefore imho `s/t/termin/` here.